persistence/src/commonTest/kotlin/com/wire/kalium/persistence/dao/ConversationDAOTest.kt

```diff
@@ -1810,6 +1810,22 @@ class ConversationDAOTest : BaseDatabaseTest() {
         )
     }
 
+    @Test
+    fun givenSubConversationGroupIdForCall_whenGettingConversationByGroupId_thenReturnNull() = runTest {
```

Comment:
```suggestion
    fun givenInsertedConversations_whenGettingConversationByInexistingGroupId_thenReturnNull() = runTest {
```

The DB has no idea what "subconversation" means. So might as well keep it generic insteaed of over-specific.